app/context/OrbisContext.tsx

```diff
@@ -84,8 +84,8 @@ export const ODB = ({ children }: OrbisDBProps) => {
 
         if (
           sessionAddress !== walletClient.account.address.toLowerCase() ||
-          //@ts-ignore
-          expTime > Date.now()
+          expTime === undefined ||
+          Date.parse(expTime) < Date.now()
```

Comment:
Can we instead do something like
```suggestion
         ( expTime !== undefined &&
          Date.parse(expTime) < Date.now()
```

Otherwise it reads a bit weird i.e. "if expTime is undefined we should remove the session entry"

Reply:
Side note: this seems like a big bug - good catch.

Reply:
Yeah, that's better. Done ✅